server/monitor.go

```diff
@@ -1764,6 +1765,9 @@ func (s *Server) Leafz(opts *LeafzOptions) (*Leafz, error) {
 	if len(s.leafs) > 0 {
 		lconns = make([]*client, 0, len(s.leafs))
 		for _, ln := range s.leafs {
+			if opts != nil && opts.Account != "" && ln.acc.Name != opts.Account {
```

Comment:
I think we should lock `ln` to access its `acc` no? (can it be a race with config reload?).
```
if opts != nil && opts.Account != "" {
    ln.mu.Lock()
    ok := ln.acc.Name == opts.Account
    ln.mu.Unlock()
    if !ok {
        continue
    }
}
```

Reply:
agreed, will change that

Reply:
fixed